### DISCUSSION TITLE
api: get jobs list

### DISCUSSION THREAD

**Discussion by jgarec** [mend/renovate-ce-ee#393] [Ideas]
Hello,

It would be interesting for us to get the list of jobs that renovate handle which is almost the same as the content of repos table in sqlite database.

Some use cases :
* check the autodiscover filter result
* check if a new project is known, if not we can use the sync api 

BR
Jérôme

**Comment by Gabriel-Ladzaretti** [comment]
An API workaround to figure out if a single repo is installed or not is using the `api/job/add`[^1] endpoint.
If the repo doesnt exist in the db, the api will return a 4xx status code.

[^1]: https://github.com/mend/renovate-ce-ee/blob/main/docs/api.md
